File: util/test/src/main/java/top/bettercode/summer/util/test/BaseLogTest.java
package top.bettercode.summer.util.test;

import ch.qos.logback.classic.Level;
import ch.qos.logback.classic.LoggerContext;
import org.junit.jupiter.api.AfterAll;
import org.slf4j.LoggerFactory;

public abstract class BaseLogTest {

  @AfterAll
  static void logAfterAll() {
    ((LoggerContext) LoggerFactory.getILoggerFactory()).getLogger(
        "org.hibernate.SQL").setLevel(
        Level.OFF);
  }

}

File: test/src/main/kotlin/top/bettercode/summer/test/BaseLogTest.kt
package top.bettercode.summer.test

import ch.qos.logback.classic.Level
import ch.qos.logback.classic.LoggerContext
import org.junit.jupiter.api.AfterAll
import org.slf4j.LoggerFactory

abstract class BaseLogTest {
    companion object {
        @JvmStatic
        @AfterAll
        fun logAfterAll() {
            (LoggerFactory.getILoggerFactory() as LoggerContext).getLogger(
                    "org.hibernate.SQL").level = Level.OFF
        }
    }
}
